block_to_imp.c

```diff
@@ -162,6 +167,8 @@ static struct trampoline_set *alloc_trampolines(char *start, char *end)
 	struct trampoline_set *metadata = calloc(1, sizeof(struct trampoline_set));
 #if _WIN32
 	metadata->buffers = VirtualAlloc(NULL, sizeof(struct trampoline_buffers), MEM_COMMIT | MEM_RESERVE, PAGE_READWRITE);
+#elif defined(HAVE_MAP_ANONYMOUS)
```

Comment:
I don't think that we need this conditional.  `posix_memalign` is likely to be less portable than `mmap`, so there's little value in having it as a fallback.